test/hydration-test.js

```diff
@@ -119,59 +147,13 @@ describe('Hydration', function() {
 
   it('should find the water intake for a user on a specified date', function() {
     expect(hydration.calculateDailyOunces(1, "2019/06/15")).to.equal(37);
-    expect(hydration.calculateDailyOunces(4, "2019/04/15")).to.equal(36);
   });
 
   it('should find water intake by day for first week', function() {
-    const user3 = new User({
```

Comment:
Nice cleanup of repetitive variables. The test file looks better now! We need to fix white scapes in userrepo-test file to be consistent